src/test/daily_mode.test.ts

```diff
@@ -5,6 +5,7 @@ import { Moves } from "#app/enums/moves";
 import { Biome } from "#app/enums/biome";
 import { Mode } from "#app/ui/ui";
 import ModifierSelectUiHandler from "#app/ui/modifier-select-ui-handler";
+import { Species } from "#app/enums/species";
```

Comment:
```suggestion
import { Species } from "#enums/species";
```
I wish VSCode would be smarter about imports...